code/modules/projectiles/gun_attachables.dm

```diff
@@ -1564,6 +1564,11 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	icon_state = "flame_wide_red"
 	range_modifier = 0
 
+/obj/item/weapon/gun/rifle/pepperball/pepperball_mini
```

Comment:
ok so let me explain this. You are defining the gun a second time. in gun_attachables. You dont want to do this. Put the attach delay underneathe the gun and remove this.